charts/navi-attractor/values.yaml

```
@@ -0,0 +1,356 @@
+# @param attractor.rtr.enabled Enable real time restrictions.
+# @param attractor.rtr.url URL real time restrictions server.
+# @param attractor.rtr.updatePeriod Update period from real time restrictions server.
+# @param attractor.mock If mock service being deployed. For test purposes only.
+# @param attractor.tilesMetricsThreshold The value at which we send tiles metrics (used for internal tests).
+# @skip rules
+
+attractor:
+  appPort: 8080
+  handlersNumber: 20
+  maxProcessTime: 20
+  responseTimelimit: 120
+  requestTimeout: 120
+  dump:
+    result: true
+    query: true
+    answer: false
+  logLevel: Info
+  logMessageField: custom.navi_msg
+  grpcPort: 50051
+  indexFilename: index.json.zip
+  citiesFilename: cities.conf.zip
+  indices:
+    # items in alphabet order
+    carOptions:
+      enabled: true
+    countryCarOptions:
+      enabled: true
+    ctxOptions:
+      enabled: true
+    entrances:
+      enabled: true
+    geoImport:
+      enabled: false
+    passZones:
+      enabled: true
+    projectCarOptions:
+      enabled: true
+    snImport:
+      enabled: false
+  attractor: {}
+  overrideConfig: ''
+  rtr:
+    enabled: false
+    url: http://rtr
+    updatePeriod: 60
+  mock: false
+  tilesMetricsThreshold: 0
+
+rules: []
+
+# @section Kafka settings for interacting with Distance Matrix Async Service
+
+# @param kafka.enabled If the Kafka is enabled
+# @param kafka.groupId Navi-Attractor service group identifier
+# @param kafka.handlersNumber Number of Kafka handlers
+# @extra kafka.properties Properties as supported by librdkafka. Refer to inline comments for details
+# @skip kafka.properties.bootstrap.servers
+# @skip kafka.properties.security.protocol
+# @param kafka.fileProperties As kafka.properties, but kept in a file, which passed to application as a filename. Refer to inline comments for details
+# @extra kafka.distanceMatrix **Settings for interacting with Distance Matrix Async service**
+# @param kafka.distanceMatrix.taskTopic Name of the topic for receiving new tasks from Distance Matrix Async API
+# @param kafka.distanceMatrix.cancelTopic Name of the topic for canceling or receiving information about finished tasks
+# @param kafka.distanceMatrix.statusTopic Name of the topic for receiving task status information
+# @param kafka.distanceMatrix.updateTaskStatusPeriodSec Update period for task statuses
+# @param kafka.distanceMatrix.messageExpiredPeriodSec Update period for task cancellations
+# @param kafka.distanceMatrix.requestDownloadTimeoutSec Timeout for downloading request data
+# @param kafka.distanceMatrix.responseUploadTimeoutSec Timeout for uploading response data
+
+kafka:
+  enabled: false
+  groupId: navi_attractor
+  handlersNumber: 2
+  properties:
+    bootstrap.servers: kafka.host:9092
+    security.protocol: PLAINTEXT
+  fileProperties: {}
+  distanceMatrix:
+    taskTopic: task_topic
+    cancelTopic: cancel_topic
+    statusTopic: status_topic
```

нужно убрать дефолт и сделать обязательными параметрами
```suggestion
  enabled: false
  groupId: ''
  handlersNumber: 2
  properties:
    bootstrap.servers: ''
    security.protocol: PLAINTEXT
  fileProperties: {}
  distanceMatrix:
    taskTopic: ''
    cancelTopic: ''
    statusTopic: ''
```

---

`group_id` дефолтный правильный, его оставил